caiman/utils/utils.py

```diff
@@ -353,7 +353,7 @@ def cell_magic_wand_wrapper(params):
 #%% From https://codereview.stackexchange.com/questions/120802/recursively-save-python-dictionaries-to-hdf5-files-using-h5py
 
 
-def save_dict_to_hdf5(dic:Dict, filename:str) -> None:
+def save_dict_to_hdf5(dic:Dict, filename:str, subdir:str='/') -> None:
```

Comment:
Wondering if subdir is necessary - os.path.join() does the right thing in a cross-platform way

Reply:
@pgunn There might be an issue there, although I just wrote a test for it that passed. Take a look?

Reply:
Actually, I'm totally wrong on this - I misread what it's doing. These are paths *inside* the hdf5 file, not real filesystem paths. os.path.join() is for filesystem paths.